### PR TITLE
CRUD functionality

### PR DESCRIPTION
**Pull request by hendrikebbers**
Some first classes for generic crud support.

---
This change is [Reviewable](https://reviewable.io/reviews/canoo/dolphin-platform/850)

### CODE REVIEW

**Comment by hendrikebbers** [comment]
@kunsingh  @giftkugel please review :)

File: platform/dolphin-platform-remoting-server/src/main/java/com/canoo/dp/impl/server/context/DolphinContext.java [hunk 1]
```diff
@@ -289,7 +291,7 @@ protected void onLongPoll() {
         }
         final Metric metric = ServerTimingFilter.getCurrentTiming().start("TaskExecution", "Execution of Tasks in Long Poll");
         try {
-            taskQueue.executeTasks();
+        taskQueue.executeTasks();
```
Comment (giftkugel) [inline platform/dolphin-platform-remoting-server/src/main/java/com/canoo/dp/impl/server/context/DolphinContext.java]: Formatting :(

File: platform/dolphin-platform-remoting-server/src/main/java/com/canoo/dp/impl/server/context/DolphinContext.java [hunk 2]
```diff
@@ -298,7 +300,7 @@ protected void onLongPoll() {
     private void onGarbageCollection() {
         final Metric metric = ServerTimingFilter.getCurrentTiming().start("RemotingGc", "Garbage collection for the remoting model");
         try {
-            garbageCollector.gc();
+        garbageCollector.gc();
```
Comment (giftkugel) [inline platform/dolphin-platform-remoting-server/src/main/java/com/canoo/dp/impl/server/context/DolphinContext.java]: Formatting :(

File: platform/dolphin-platform-remoting-server/src/main/java/com/canoo/dp/impl/server/context/DolphinContext.java [hunk 3]
```diff
@@ -321,12 +323,17 @@ public String getId() {
     }
 
     public List<Command> handle(final List<Command> commands) {
+        active = true;
+        try {
         final List<Command> results = new LinkedList<>();
-        for (Command command : commands) {
-            results.addAll(serverConnector.receive(command));
-            hasResponseCommands = !results.isEmpty();
+            for (Command command : commands) {
```
Comment (giftkugel) [inline platform/dolphin-platform-remoting-server/src/main/java/com/canoo/dp/impl/server/context/DolphinContext.java]: Add final keyword.

File: platform/dolphin-platform-remoting-server/src/main/java/com/canoo/dp/impl/server/context/DolphinContext.java (hunk 1 above)
Comment (hendrikebbers) [reply platform/dolphin-platform-remoting-server/src/main/java/com/canoo/dp/impl/server/context/DolphinContext.java]: done

File: platform/dolphin-platform-remoting-server/src/main/java/com/canoo/dp/impl/server/context/DolphinContext.java (hunk 2 above)
Comment (hendrikebbers) [reply platform/dolphin-platform-remoting-server/src/main/java/com/canoo/dp/impl/server/context/DolphinContext.java]: Done

File: platform/dolphin-platform-remoting-server/src/main/java/com/canoo/dp/impl/server/context/DolphinContext.java (hunk 3 above)
Comment (hendrikebbers) [reply platform/dolphin-platform-remoting-server/src/main/java/com/canoo/dp/impl/server/context/DolphinContext.java]: done

**Pull request merged**